community/kernel/src/main/java/org/neo4j/kernel/impl/store/id/IdContainer.java

```diff
@@ -314,6 +314,22 @@ public static void createEmptyIdFile( FileSystemAbstraction fs, File file, long
         }
     }
 
+    private static int readBuffer( StoreChannel channel, ByteBuffer buffer ) throws IOException
```

Comment:
Does it make sense for `StoreChannel` to have `readAll` the same way it has `writeAll` instead of having this static method here?